packages/web-components/src/kumo/spinner/spinner.stories.ts

```diff
@@ -0,0 +1,84 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../../helpers.stories.js';
+import { SpinnerAppearance, SpinnerSize } from '../../spinner/spinner.options.js';
+
+type SpinnerStoryArgs = Args;
+type SpinnerStoryMeta = Meta<SpinnerStoryArgs>;
+
+const storyTemplate = html<SpinnerStoryArgs>`
+  <kumo-spinner appearance=${x => x.appearance} size=${x => x.size}></kumo-spinner>
```

Comment:
Will kumo have an appearance attribute?

Reply:
And size?

Reply:
Currently yes, both of those are reflected in the comps. There's also a label feature which is coming from [Fluent React's Spinner](https://react.fluentui.dev/?path=/docs/components-spinner--default), which we don't currently have... but could build either Fluent or Kumo to parity there, it would just require more DIVs.